Utilities/IntervalFiles/README.md

```diff
@@ -0,0 +1,237 @@
+use any subset of them when running a benchmarking tool on Terra.
+
+```
+strat_intervals = [
+"gs://broad-dsde-methods-hydro-gen-truth-data-public/NIST/IntervalFiles/GRCh38_gc85_slop50.bed.gz",
+"gs://broad-dsde-methods-hydro-gen-truth-data-public/NIST/IntervalFiles/GRCh38_gc15_slop50.bed.gz",
+"gs://broad-dsde-methods-hydro-gen-truth-data-public/NIST/IntervalFiles/GRCh38_lowmappabilityall.bed.gz",
+"gs://broad-dsde-methods-hydro-gen-truth-data-public/NIST/IntervalFiles/GRCh38_AllHomopolymers_gt6bp_imperfectgt10bp_slop5.bed.gz"
+"gs://broad-dsde-methods-hydro-gen-truth-data-public/NIST/IntervalFiles/LCRFromHengHg38.bed",
+"gs://broad-dsde-methods-hydro-gen-truth-data-public/NIST/IntervalFiles/GRCh38_segdups.bed.gz",
+"gs://broad-dsde-methods-hydro-gen-truth-data-public/NIST/IntervalFiles/GRCh38_alldifficultregions.bed.gz",
+"gs://gcp-public-data--broad-references/hg38/v0/exome_evaluation_regions.v1.interval_list",
+"gs://broad-dsde-methods-hydro-gen-truth-data-public/NIST/IntervalFiles/GRCh38_refseq_cds.bed.gz",
+"gs://broad-dsde-methods-hydro-gen-truth-data-public/NIST/IntervalFiles/GRCh38_BadPromoters.bed.gz",
+"gs://gcp-public-data--broad-references/hg38/v0/ultima_genomics/interval_lists/ug_hcr.bed",
+]
+
+strat_labels = [
+"HighGC",
+"LowGC",
+"LowMappability",
+"Homopolymers",
+"LowComplexityRegion",
+"SegDup",
+"AllDifficult",
+"Exome",
+"CodingRegions",
+"BadPromoters",
+"UGHiConf",
+]
+```
+
+## Appendix on Statistics
+
+Above, we compute the amount of redundancy between two interval lists using the [Jaccard index](https://en.wikipedia.org/wiki/Jaccard_index).
+Recall for two sets A and B, this is given by the size of A intersect B divided by the size of A union B. Note this is bounded
+by 0 and 1. It is equal to 1 if and only if the two sets are identical, and equal to 0 if and only if they are disjoint.
+
+We compute this statistic on a few "interesting" pairs and report it above, along with asymmetric redundancy calculations
+measuring what percentage of A lies in B. This is the special case of the Jaccard index of A intersect B with A. Note also
+the percent of genome covered can be computed using a Jaccard index of your bed file A with a "full" bed file covering
+all contigs of your genome. A script to make a "full" bed from a reference dict is included in the `create_full_bed.sh`
+file. In the above calculations, a "full" bed was created from hg38 for just the autosomes and X/Y to compute what 
+percent of the genome a given interval file spanned.
+
+A script to compute all three of these (the usual Jaccard index, percent of A in B, and percent of B in A) for two 
+input bed files A and B is provided in the `overlap_stats.sh` file for reproducibility of the above code and extension 
+beyond the provided cases. You must have the `hg38-noalt.genome` or similar file for your reference in the same directory 
+when running this script for it to function properly, which is an ordered list of contigs with sequence length. Overlap
```

Comment:
Did you remove N's from the sequence length? I think it's fine either way, but it would be good to make it clear here how you got the sequence length. 

The reason I noticed this is because I think you're estimate of the percentage of the genome that the UGHiConf interval list is lower than what they report. I think this happens because they are careful to make the denominator smaller in reasonable ways. I think either way of reporting it is fine as long as it's clear.

Reply:
Those stats were computed without removing N bases, but I added a comment to help clarify this. I did a little testing with an interval list without the Ns, and in particular, the UG one is closer to the expected 95% or so without them.